pydicom/dataset.py

```diff
@@ -820,7 +843,7 @@ def RemoveCallback(dataset, data_element):
 
         self.walk(RemoveCallback)
 
-    def save_as(self, filename, write_like_original=True):
+    def save_as(self, filename, write_like_original=False):
```

Comment:
I'm not sure about this one - I think this has been True at least since 0.98, have to check this tomorrow.

Reply:
It's possible,  my application has been shipping with a modified version of 0.9.9,  so I may have changed that locally long ago...

Reply:
You are correct, this parameter has always been True.  write_file_meta_info in filewriter.py now has a new parameter "enforce_standard=True"  that is changing the old behaviour. write_file_meta_info always used to fill in the FileMetaInformationGroupLength and the FileMetaInformationVersion.  I will revert this and await some more comments on why the write_file_meta_info behaviour was changed.  I think the original behaviour should be preserved as using `dataset.save_as("file", write_like_original=False)` in the client code just seems unpleasant.